### PR TITLE
Edge/edge release notes 1011

### PR DESCRIPTION
**Pull request by sharath-jk**

### CODE REVIEW

**Review by MWindel — APPROVED** [review]

**Comment by BeateRixen** [comment]
Please add the new release version variable to all links like in the following notation:

https://cumulocity.com/guides{{< 10-11-0 >}}/edge/configuration/#changing-domain-name

This helps us to always link to the correct version.

File: content/release-10-11-0/edge-10-11-0-bundle/10_11_0.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+Download the Cumulocity IoT Edge version 10.11 from [Software AG Empower portal](https://empower.softwareag.com), based on the target hypervisor.
+
+### Changing the domain name
+
+You can now change the domain name of your Edge appliance using the user interface or REST API. For more information, see [Changing the domain name](https://cumulocity.com/guides/edge/configuration/#changing-domain-name).
+
+### Remotely update the Edge appliance
+
+You can now update the Edge appliance remotely from a tenant account. To use this feature, you must have enabled remote-connectivity in your Edge appliance. For more information, see [Updating Edge remotely](https://cumulocity.com/guides/edge/update/#updating-edge-remotely).
```
Comment (BeateRixen) [inline content/release-10-11-0/edge-10-11-0-bundle/10_11_0.md:17]: enabled remote-connectivity > remote-connectivity enabled ?

File: content/release-10-11-0/edge-10-11-0-bundle/10_11_0.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+### Remotely restart the Edge appliance
+
+You can now restart the Edge appliance remotely from a tenant account. To use this feature, you must have enabled remote-connectivity in your Edge appliance. For more information, see [Restarting Edge remotely](https://cumulocity.com/guides/edge/remote-connectivity/#updating-edge-remotely).
```
Comment (BeateRixen) [inline content/release-10-11-0/edge-10-11-0-bundle/10_11_0.md:21]: enabled remote-connectivity > see above

File: content/release-10-11-0/edge-10-11-0-bundle/10_11_0.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+### User interface enhancements for configuration
+
+You can now use the user interface to:
+
+- [Enable the microservice hosting feature](https://cumulocity.com/guides/edge/configuration/#enabling-or-disabling-the-microservice-hosting-feature-using-the-ui)
+- [Expand the disk size](https://cumulocity.com/guides/edge/configuration/#expanding-the-disk-size)
+- [Restart the Edge appliance](https://cumulocity.com/guides/edge/configuration/#edge-reboot)
+
+For new enhancements in Cumulocity IoT, see [What's new](/release-10-9-0/whatsnew-10-9-0/).
```
Comment (BeateRixen) [inline content/release-10-11-0/edge-10-11-0-bundle/10_11_0.md:31]: Change the link to point to 10.11

File: content/release-10-11-0/edge-10-11-0-bundle/10_11_0.md (hunk 1 above)
Comment (sharath-jk) [reply content/release-10-11-0/edge-10-11-0-bundle/10_11_0.md:17]: Done

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**